FF1Lib/FF1Rom.cs

```diff
@@ -963,10 +975,10 @@ public void UpgradeToMMC3()
 		public void WriteSeedAndFlags(string version, string seed, string flags)
 		{
 			// Replace most of the old copyright string printing with a JSR to a LongJump
-			Put(0x38486, Blob.FromHex("20FCFE60"));
+			Put(0x38486, Blob.FromHex("20B9FF60"));
 
 			// DrawSeedAndFlags LongJump
-			PutInBank(0x1F, 0xFEFC, CreateLongJumpTableEntry(0x0F, 0x8980));
+			PutInBank(0x1F, 0xFFB9, CreateLongJumpTableEntry(0x0F, 0x8980));

```

Comment:
is there a reason this was moved? I didnt believe there was much room left for moving these longjumps around

-- I really need to get around to fixing that

Reply:
CHRload isn't very longjump friendly, and there's just enough free space there to put the new routine without the DrawSeedAndFlags long jump, so I moved it to a smaller corner of free space.